features/support/database_cleaner.rb

```diff
@@ -1,18 +0,0 @@
-begin
```

Comment:
We now have  which https://github.com/grosser/test_after_commit should make these changes unneeded.